### ISSUE TITLE
MacOS installation script/routine

### ISSUE THREAD

**Issue by davidpablocohn** [OceanDataTools/openrvdas#120]
OpenRVDAS will mostly be used on Linux, but a lot of people develop on MacOS, so it would be very handy to have a MacOS installation script that mirrors the Linux scripts in openrvdas/utils.

**Comment by davidpablocohn** [comment]
Not quite a script yet, but these steps get OpenRVDAS running enough for the local runserver command to work:
```
RVDAS_USER=rvdas
RVDAS_PASSWORD=rvdas

#Install git:
git --version  # prompts for installation of command line tools

#Install homebrew:
ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"

#Install python3
brew install python socat mysql openssh

brew tap homebrew/services
brew services start mysql
mysql_secure_installation

# set up tables in MySQL (copied from build script)
echo Please enter MySQL root password again to continue configuration:
mysql -u root -p <<EOF 
drop user if exists 'test'@'localhost'; 
create user 'test'@'localhost' identified by 'test';

drop user if exists 'rvdas'@'localhost';
create user '$RVDAS_USER'@'localhost' identified by '$RVDAS_PASSWORD';

create database if not exists data character set utf8;
GRANT ALL PRIVILEGES ON data.* TO '$RVDAS_USER'@'localhost';

create database if not exists openrvdas character set utf8;
GRANT ALL PRIVILEGES ON openrvdas.* TO '$RVDAS_USER'@'localhost';

create database if not exists test character set utf8;
GRANT ALL PRIVILEGES ON test.* TO '$RVDAS_USER'@'localhost';
GRANT ALL PRIVILEGES ON test.* TO 'test'@'localhost' identified by 'test';

flush privileges;
\q
EOF
echo Done setting up database

pip3 install Django==2.0 pyserial uwsgi websockets PyYAML \
             parse mysqlclient mysql-connector

git clone https://github.com/davidpablocohn/openrvdas.git

#add hostname to /etc/hosts
```

At that point, you can open three terminals and check that it's running:
- Terminal 1 (set up serial ports): 
  ``logger/utils/simulate_serial.py --config test/nmea/NBP1406/serial_sim_NBP1406.yaml --loop``
- Terminal 2 (run local server):
  ``./manage.py runserver localhost:8000``
- Terminal 3 (run logger manager):
  ``server/logger_manager.py --database django --start_data_server --config test/NBP1406/NBP1406_cruise.yaml --mode monitor``

Then open a browser to ``http://localhost:8000`` to see the control console and ``http://localhost:8000/static/widgets/nbp_demo.html`` to see the sample widget page.

To install it so that it can use NGINX or some other webserver would require a bunch more work, I think, including configuring uwsgi, which has always been a mystery to me. But this seems like a good start.

**Comment by davidpablocohn** [comment]
Resolved as of f86a6fd7179f75c49fd94d1e3949785d9dfe680b with merge from branch issue_120
